docs/README.md

```diff
@@ -0,0 +1,71 @@
+# Manifestor specification
+
+This is the current specification for Manifestor manifests.
+
+```yaml
+# top-level "environments" a list of environments, probably representing
+# namespaces, possibly clusters.
+environments:
+  # this is an environment called "development"
+  - name: development
+    # "pipelines" is a property of the environment, this represents the
+    # pipeline bindings to be used in the automatically generated EventListener.
+    # If a service has a `source_url` then we can automatically trigger these
+    # pipeline bindings based on Git Hosting Service hooks.
+    # 
+    # The "integration" pipeline would be triggered on PullRequest events.
+    # The "deployment" pipeline would be triggered on Push events.
+    pipelines:
+      integration:
+        template: dev-ci-template
+        binding: dev-ci-binding
+      deployment:
+        template: dev-cd-template
+        binding: dev-cd-binding
+    # This is a set of "apps", which according to the IBM/Red Hat repository
+    # "contain" a set of services.
+    apps:
+      - name: my-app-1
+        services:
+          - name: app-1-service-http
+            # the "source_url" represents the "upstream" source code e.g. the
+            # Java/Go/Node.js code.
+            #
+            # This is used when generating the EventListener to drive the CI
+            # process, PullRequests and Pushes to this branch drive the
+            # pipelines.
+            source_url: https://github.com/myproject/myservice.git
+          - name: app-1-service-metrics
```

Comment:
For those with no `source_url`, it implies that the service is not to be built off a repo, rather all kube manifests would be present in the 'gitops' repo?

Reply:
No, it means that we wouldn't include it in the eventlistener...

Reply:
That's not to say that it's not more generally useful for documentation...